tern/command_lib/snippets.yml

```diff
@@ -121,3 +121,9 @@ yarn:
     - 'add'
     - 'remove'
   packages: 'npm'
+zypper:
+  install: 'in'
```

Comment:
Now that https://github.com/tern-tools/tern/pull/716 has been merged, can you rebase this PR and update the install list to include `install` and `in` ? Same for remove -- you can list `rm` and `remove`.

Reply:
Done!